src/test/java/codeu/controller/RegisterServletTest.java

```diff
@@ -0,0 +1,36 @@
+package codeu.controller;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class RegisterServletTest 
+{
+	private RegisterServlet registerServlet;
```

Comment:
One tiny thing: the official style guide we're supposed to be following says to use spaces instead of tabs: https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation

Can you replace each tab with two spaces? This will make our lives easier as we write more complicated code, and will be good to be standard throughout the codebase.

(I personally prefer to use tabs as well, so if this is annoying I feel your pain!)

Reply:
yes! i'll adjust that now, sorry!